xwiki-rendering-api/src/main/java/org/xwiki/rendering/internal/transformation/DefaultRenderingContext.java

```diff
@@ -202,7 +203,7 @@ public void transformInContext(Transformation transformation, TransformationCont
             return stack;
         }
 
-        return null;
+        return Collections.emptyList();
```

Comment:
As you can see from the using it  it's expected that this method might return null. Also this code won't build.